addons/purchase/models/purchase.py

```diff
@@ -192,6 +193,21 @@ def create(self, vals):
         return super(PurchaseOrder, self).create(vals)
 
     @api.multi
+    def write(self, values):
```

Comment:
@sle-odoo We had perfect code before on the level of the purchase order line before (v10), but maybe @pimodoo considered it not necessary and was already happy with the next activity.  What if you change the PO line and it does not pass through the PO (which I thought it normally did)?

Reply:
[1] Ice Cream:
Ordered Quantity: 1.0 -> 3.0 
Received Quantity: 1.0 
Billed Quantity: 0.0